sbt/src/sbt-test/dependency-management/circular-dependency/changes/multi.sbt

```diff
@@ -4,6 +4,7 @@ def commonSettings: Seq[Def.Setting[_]] =
   Seq(
     ivyPaths := new IvyPaths( (baseDirectory in ThisBuild).value, Some((target in LocalRootProject).value / "ivy-cache")),
     scalaVersion := "2.10.4",
+    fullResolvers := fullResolvers.value.filterNot(_.name == "inter-project"),
```

Comment:
You need this because sbt itself will cause problems with circular dependencies right?